From 092770e3fc19cdf3b2e84de09cb6139b15096045 Mon Sep 17 00:00:00 2001
From: Zongchun Yu <Zongchun.Yu@freescale.com>
Date: Mon, 14 Apr 2014 10:52:05 +0800
Subject: [PATCH] Fix compile issue

Fix:
| powerpc-fsl-linux-gcc   -DCONFIG_GACT -DCONFIG_GACT_PROB -Wl,-export-dynamic -I../include/ -shared -fpic -o q_ceetm.so q_ceetm.c
| q_ceetm.c:15:19: fatal error: stdio.h: No such file or directory
| compilation terminated.
| make[1]: *** [q_ceetm.so] Error 1

Signed-off-by: Zongchun Yu <Zongchun.Yu@freescale.com>
---
 usr/lib/Makefile |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/usr/lib/Makefile b/usr/lib/Makefile
index ae233a4..77bc204 100644
--- a/usr/lib/Makefile
+++ b/usr/lib/Makefile
@@ -13,7 +13,11 @@ ARCH = powerpc
 ifeq ($(ARCH), x86_64)
 CFLAGS = -DX86_64 -D__LP64__
 else
-CFLAGS =
+ifdef SYSROOT
+  CFLAGS = --sysroot=$(SYSROOT)
+else
+  CFLAGS =
+endif
 endif
 
 SHARED_LIBS ?= y
-- 
1.7.0.4

